packages/flutter_tts/tizen/inc/flutter_tts_tizen_plugin.h

```diff
@@ -1,3 +1,7 @@
+// Copyright 2021 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
```

Comment:
No, in general.

Reply:
@swift-kim OK, should the license notice be added to the header files of other packages as well?

Reply:
?

Reply:
There is no license notice in the automatically generated header file. 
as the license notice is added in this PR, should we add license notice to other packages? or should we remove this fix from this PR?

Reply:
> should we remove this fix from this PR?

Yes, I think.

Reply:
Thanks you for answer.
CC @JSUYA

Reply:
I will remove this part.
And I did the same thing when refactoring the geolocator_tizen package.
https://github.com/flutter-tizen/plugins/pull/410 PR is fix it.

Reply:
@bbrto21 Basically because no portion of the code is written by the specified copyright holder. If you want to add the information to the plugin header file, the same thing should be added to the `log.h` file for consistency. I don't think that's what you want.

Reply:
Thank you :) It's just that I haven't figured out how to explain this to newcomers.

Reply:
Oh I have a question related to this.
Should I remove the license notice from XXX_plugin.cc and log.h as well?

```shell
~/dev/os/f-project/plugins (dev/flutter_tts_refactor) 18:09:37 $ grep -nr "Copyright 202" ./ | grep "log.h"
./packages/tizen_audio_manager/tizen/src/log.h:  1:	// Copyright 2022 Samsung Electronics Co., Ltd. All rights reserved.

~/dev/os/f-project/plugins (dev/flutter_tts_refactor) 18:09:40 $ grep -nr "Copyright 202" ./ | grep "plugin.h"
./packages/tizen_audio_manager/tizen/inc/tizen_audio_manager_plugin.h:  1:	// Copyright 2022 Samsung Electronics Co., Ltd. All rights reserved.
./packages/messageport/tizen/inc/messageport_tizen_plugin.h:  1:	// Copyright 2021 Samsung Electronics Co., Ltd. All rights reserved.
./packages/tizen_package_manager/tizen/inc/tizen_package_manager_plugin.h:  1:	// Copyright 2022 Samsung Electronics Co., Ltd. All rights reserved.
./packages/geolocator/tizen/inc/geolocator_tizen_plugin.h:  1:	// Copyright 2021 Samsung Electronics Co., Ltd. All rights reserved.
./packages/battery_plus/tizen/inc/battery_plus_tizen_plugin.h:  1:	// Copyright 2020 Samsung Electronics Co., Ltd. All rights reserved.
./packages/tizen_app_manager/tizen/inc/tizen_app_manager_plugin.h:  1:	// Copyright 2022 Samsung Electronics Co., Ltd. All rights reserved.


~/dev/os/f-project/plugins (dev/flutter_tts_refactor) 18:09:46 $ grep -nr "Copyright 202" ./ | grep "plugin.cc"
./packages/device_info_plus/tizen/src/device_info_plus_tizen_plugin.cc:   1:	// Copyright 2021 Samsung Electronics Co., Ltd. All rights reserved.
./packages/permission_handler/tizen/src/permission_handler_tizen_plugin.cc:   1:	// Copyright 2021 Samsung Electronics Co., Ltd. All rights reserved.
./packages/sensors_plus/tizen/src/sensors_plus_plugin.cc:   1:	// Copyright 2021 Samsung Electronics Co., Ltd. All rights reserved.
./packages/tizen_audio_manager/tizen/src/tizen_audio_manager_plugin.cc:    1:	// Copyright 2022 Samsung Electronics Co., Ltd. All rights reserved.
./packages/messageport/tizen/src/messageport_tizen_plugin.cc:   1:	// Copyright 2021 Samsung Electronics Co., Ltd. All rights reserved.
./packages/webview_flutter/tizen/src/webview_flutter_tizen_plugin.cc:   1:	// Copyright 2021 Samsung Electronics Co., Ltd. All rights reserved.
./packages/image_picker/tizen/src/image_picker_tizen_plugin.cc:   1:	// Copyright 2021 Samsung Electronics Co., Ltd. All rights reserved.
./packages/tizen_package_manager/tizen/src/tizen_package_manager_plugin.cc:    1:	// Copyright 2022 Samsung Electronics Co., Ltd. All rights reserved.
./packages/geolocator/tizen/src/geolocator_tizen_plugin.cc:   1:	// Copyright 2021 Samsung Electronics Co., Ltd. All rights reserved.
./packages/flutter_tts/tizen/src/flutter_tts_tizen_plugin.cc:    1:	// Copyright 2021 Samsung Electronics Co., Ltd. All rights reserved.
./packages/battery_plus/tizen/src/battery_plus_tizen_plugin.cc:   1:	// Copyright 2020 Samsung Electronics Co., Ltd. All rights reserved.
./packages/network_info_plus/tizen/src/network_info_plus_tizen_plugin.cc:   1:	// Copyright 2021 Samsung Electronics Co., Ltd. All rights reserved.
./packages/audioplayers/tizen/src/audioplayers_tizen_plugin.cc:    1:	// Copyright 2021 Samsung Electronics Co., Ltd. All rights reserved.
./packages/tizen_notification/tizen/src/tizen_notification_plugin.cc:    1:	// Copyright 2021 Samsung Electronics Co., Ltd. All rights reserved.
./packages/sqflite/tizen/src/sqflite_plugin.cc:    1:	// Copyright 2021 Samsung Electronics Co., Ltd. All rights reserved.
./packages/wearable_rotary/tizen/src/wearable_rotary_plugin.cc:   1:	// Copyright 2021 Samsung Electronics Co., Ltd. All rights reserved.
./packages/tizen_app_manager/tizen/src/tizen_app_manager_plugin.cc:    1:	// Copyright 2022 Samsung Electronics Co., Ltd. All rights reserved.
./packages/wakelock/tizen/src/wakelock_tizen_plugin.cc:   1:	// Copyright 2021 Samsung Electronics Co., Ltd. All rights reserved.
./packages/package_info_plus/tizen/src/package_info_plus_tizen_plugin.cc:   1:	// Copyright 2021 Samsung Electronics Co., Ltd. All rights reserved.
./packages/video_player/tizen/src/video_player_tizen_plugin.cc:   1:	// Copyright 2020 Samsung Electronics Co., Ltd. All rights reserved.
./packages/integration_test/tizen/src/integration_test_plugin.cc:   1:	// Copyright 2020 Samsung Electronics Co., Ltd. All rights reserved.
./packages/connectivity_plus/tizen/src/connectivity_plus_tizen_plugin.cc:   1:	// Copyright 2020 Samsung Electronics Co., Ltd. All rights reserved.
./packages/camera/tizen/src/camera_plugin.cc:    1:	// Copyright 2021 Samsung Electronics Co., Ltd. All rights reserved.
~/dev/os/f-project/plugins (dev/flutter_tts_refactor) 18:09:56 $ 

```

Reply:
>@bbrto21 Basically because no portion of the code is written by the specified copyright holder. If you want to add the information to the plugin header file, the same thing should be added to the log.h file for consistency. I don't think that's what you want.

In https://github.com/flutter-tizen/plugins/pull/410 , I added a commit that removes copyright from the log.